libp2p/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.56.0
+
+- Remove deprecated `Transport::with_bandwidth_logging`, `SwarmBuilder::with_bandwidth_logging` and `TransportExt`. 
+  See [PR 5766](https://github.com/libp2p/rust-libp2p/pull/5766).
+
 ## 0.55.1
 - Introduce `libp2p-webrtc-websys` behind `webrtc-websys` feature flag.
   See [PR 5819](https://github.com/libp2p/rust-libp2p/pull/5819).
```

Comment:
```suggestion
- Introduce `libp2p-webrtc-websys` behind `webrtc-websys` feature flag.
  See [PR 5819](https://github.com/libp2p/rust-libp2p/pull/5819).
- Remove deprecated `Transport::with_bandwidth_logging`, `SwarmBuilder::with_bandwidth_logging` and `TransportExt`. 
  See [PR 5766](https://github.com/libp2p/rust-libp2p/pull/5766).
```

v0.55.1 hasn't been released, so the CHANGELOG entries should be merged.

Reply:
Are we releasing 0.56 very soon?

Reply:
Depends. Is there specific need for it? 
If someone needs `libp2p-webrtc-websys` we can also just do an individual release for that crate.

Reply:
> Depends. Is there specific need for it? 

This PR? No, this one is quite low priority.

Reply:
Sorry, this was probably a misunderstanding. 
We do need to bump swarm to `0.56.0` because it's a breaking change. I meant in my comment just that the old entry for `0.55.1` just be merged with the entry of `0.56.0` because there won't be a `0.55.1` release.